PowerShell/Send-DataFile.ps1

```diff
@@ -17,3 +17,5 @@ $response = Invoke-WebRequest `
   -Headers @{ "x-api-key" = $apiKey } `
   -ContentType "multipart/form-data" `
   -Form @{ file = $file; type = "text/csv" }
+
+return $response
```

Comment:
So if this errors out (like it did beforehand), you'll get the output printed on your screen.  But if everything succeeds, it won't print anything on your screen and you're left wondering if it worked at all.  This makes it print as expected (or let you pipe the output of the script into something else)